typespec-extension/src/code-model-builder.ts

```diff
@@ -2382,6 +2392,15 @@ export class CodeModelBuilder {
     }
   }
 
+  private getConvenienceApiNameFromServiceMethod(sdkMethod: SdkServiceMethod<SdkHttpOperation>): string | undefined {
+    // check @convenienceMethod
+    if (sdkMethod.__raw && shouldGenerateConvenient(this.sdkContext, sdkMethod.__raw)) {
```

Comment:
Do we still need to do __raw here?

Reply:
Good catch. There is a but previously and I used `raw`, https://github.com/Azure/typespec-azure/issues/980, now it is fixed, I will switch to use TCGC to check `@convenientAPI`.